content/documentation/pages/6-recipes/7-scaling/0-index.md

```diff
@@ -0,0 +1,29 @@
+---
+path: 'recipes/scaling/'
+title: 'Scaling'
+description: 'Using Prometheus and Data Flow Scale API for autoscaling pipelines'
```

Comment:
> Using the Prometheus and Data Flow's Scale API for autoscaling streaming applications

Reply:
As discussed on the call, let's move this entire thing under https://dataflow.spring.io/docs/feature-guides/streams/ — perhaps right next to "Data Partitioning". We could call it "Autoscaling Streams".